releases/3.3.0/artifacts.json

```
@@ -0,0 +1,212 @@
+{
+  "3.3.0": {
+    "holoscan": {
+      "debian-version": "3.1.0.3-1",
+      "wheel-version": "3.1.0",
+      "base-images": {
+        "dgpu": "nvcr.io/nvidia/cuda:12.8.1-runtime-ubuntu24.04",
+        "igpu": "nvcr.io/nvidia/tensorrt:25.02-py3-igpu"
```

Base container reviewed to TensorRT 25.03
```suggestion
        "igpu": "nvcr.io/nvidia/tensorrt:25.03-py3-igpu"
```

---

Thank you @tbirdso